docs/design/topology_awareness.md

```diff
@@ -0,0 +1,190 @@
+    zones: int # How many zones will be used to assign pods there. It must be higher than 0.
+```
+
+Disable topology:
+```yaml
+spec:
+  topology:
+    enable: false
+```
+or remove `spec.topology` object.
+
+## How to check which ArangoDB members are assigned to the topology
+
+#### Topology aware
+
+Each member should be topology aware, and it can be checked in list of conditions here `status.members.[agents|coordinators|dbservers].conditions`. 
+Example:
+```yaml
+status:
+  ...
+  members:
+    agents:
+    - conditions:                  
+         reason:  Topology awareness enabled
+         status:  True
+         type:    TopologyAware
+```
+
+If `status` for the condition's type `TopologyAware` is set to `false` then it is required to replace ArangoDB member.
+To do so we need to set pod's annotation `deployment.arangodb.com/replace` to `true`, starting from all
+coordinators which are not assigned to any zone. This situation usually happens when 
+topology was enabled on an existing ArangoDeployment resource.
+
+#### Member topology
+Each member's status should have topology, and it can be checked here `status.members.[agents|coordinators|dbservers].topology` and here `status.topology`. 
+Example:
+```yaml
+status:
+  ...
+  members:
+    agents:
+    - id: AGNT-2shphs7a
+      topology:
+        id:     35a61527-9d2b-49df-8a31-e62417fcd7e6
+        label:  eu-central-1c
+        rack:   0
+  ...
+  topology:
+    id:     35a61527-9d2b-49df-8a31-e62417fcd7e6
+    label:  topology.kubernetes.io/zone
+    size:   3
+    zones:
+      - id: 0
+        labels:
+          - eu-central-1c
+        members:
+          agnt:
+            - AGNT-2shphs7a
+          ...
+      - ...
+  ...
+```
+which means that `AGNT-2shphs7a` is assigned to `eu-central-1c`.
+
+#### Pod's labels
+
+A pod which belongs to the member should have two new labels. 
+Example:
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    deployment.arangodb.com/topology: 35a61527-9d2b-49df-8a31-e62417fcd7e6
+    deployment.arangodb.com/zone: "0"
+```
+
+#### A pod anti affinity
+
+A pod which belongs to the member should have a new pod anti affinity rules. 
+Example:
+```yaml
+spec:
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: deployment.arangodb.com/topology
+            operator: In
+            values:
+            - 35a61527-9d2b-49df-8a31-e62417fcd7e6
+          - ...
+          - key: deployment.arangodb.com/zone
+            operator: In
+            values:
+            - "1"
+            - "2"
+          - ...
+        topologyKey: topology.kubernetes.io/zone
+      - ...
+```
+which means that pod can not be assigned to zone `1` and `2`.
+
+#### A node affinity
+
+A pod which belongs to the member can have a node affinity rules. If a pod does not have it then it means that it will have pod's affinities. 
```

Comment:
```suggestion
A pod which belongs to the member can have a node affinity rules. If a pod does not have it then it means that it will have pod's affinities. 
```
```suggestion
A pod which belongs to the member can have a node affinity rules. If a pod does not have it then it will have pod affinities. 
```